packages/mobile/src/onboarding/contacts/ImportContactsScreen.tsx

```diff
@@ -1,227 +1,187 @@
     )
   }
 
-  render() {
-    const { isFindMeSwitchChecked } = this.state
-    const {
-      t,
-      importContactsProgress: { status },
-    } = this.props
-
-    const isSkipHidden = status === ImportContactsStatus.Done
-
-    return (
-      <SafeAreaView style={styles.container}>
-        <ScrollView contentContainerStyle={styles.scrollContainer}>
-          {status <= ImportContactsStatus.Stopped && (
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView contentContainerStyle={styles.scrollContainer}>
+        <View>
+          <Text style={styles.contactHeader} testID="ImportContactsScreenHeader">
+            {t('contacts.header')}
+          </Text>
+          <Text style={styles.body}>{t('contacts.body')}</Text>
+          {importStatus <= ImportContactsStatus.Stopped && (
             <>
-              <Text style={styles.h1} testID="ImportContactsScreenHeader">
-                {t('contacts.header')}
-              </Text>
-              <Text style={styles.body}>{t('contacts.body')}</Text>
               <View style={styles.switchContainer}>
-                <Switch value={isFindMeSwitchChecked} onValueChange={this.onToggleFindMeSwitch} />
+                <Switch value={isFindMeSwitchChecked} onValueChange={onToggleFindMeSwitch} />
                 <Text style={styles.switchText}>{t('contacts.findSwitch')}</Text>
               </View>
               <Button
-                onPress={this.onPressConnect}
+                onPress={onPressConnect}
                 text={t('global:connect')}
                 size={BtnSizes.MEDIUM}
-                type={BtnTypes.SECONDARY}
+                type={BtnTypes.ONBOARDING}
               />
             </>
           )}
-          {status > ImportContactsStatus.Stopped && this.renderImportStatus()}
-        </ScrollView>
-        <View style={[styles.bottomButtonContainer, isSkipHidden && styles.invisible]}>
-          <TextButton
-            disabled={isSkipHidden}
-            onPress={this.onPressSkip}
-            style={styles.bottomButtonText}
-            testID="ImportContactsScreenSkipButton"
-          >
-            {t('global:skip')}
-          </TextButton>
         </View>
-      </SafeAreaView>
-    )
-  }
+        {importStatus > ImportContactsStatus.Stopped && (
+          <View style={styles.statusContainer}>{statusComponents}</View>
+        )}
+      </ScrollView>
+    </SafeAreaView>
+  )
 }
 
+ImportContactsScreen.navigationOptions = ({ navigation }: ScreenProps) => ({
+  ...nuxNavigationOptionsNoBackButton,
+  headerTitle: () => (
+    <HeaderTitleWithSubtitle
+      title={i18n.t('onboarding:verificationEducation.title')}
+      subTitle={i18n.t('onboarding:step', { step: '5' })}
+    />
+  ),
+  headerRight: () => (
+    <TopBarTextButton
+      title={i18n.t('global:skip')}
+      testID="VerificationEducationSkip"
+      // tslint:disable-next-line: jsx-no-lambda
+      onPress={() => navigate(Screens.OnboardingSuccessScreen)}
```

Comment:
This is missing a lot of the stuff we need to do onSkip right? Like cancelling the import

Reply:
I certainly could be wrong, but it looked to me that `denyImportContacts` wasn't actually doing anything. It triggered an action that flipped `askedContactsPermission` in redux state to `true` but as far as I can tell `askedContactsPermission` is not used anywhere. I didn't want to delete without knowing the background. Do you have insight into the context there? 

On second thought, I can see why having `cancelImportContacts` occur on skip is useful for much slower phones. I will add that back. It happens so quickly on my phone it felt useless to have but need to get better at remembering that's not likely to be the norm.

Reply:
Yeah askedContactsPermission was used in navigator but looks like thats gone now, probably during the big nav update. 
I think that's still useful to track this though in case we want to show different UI for users that saw this and skipped vs never saw it at all.

Reply:
Gotcha, I will add `denyImportContacts` back into the `onSkip` stuff with the understanding that we may make use of it sometime in the future